src/components/component.ts

```diff
@@ -589,4 +589,11 @@ module Plottable {
       return this._backgroundContainer;
     }
   }
+  export class Alignment {
+    static TOP = "top";
+    static BOTTOM = "bottom";
+    static LEFT = "left";
+    static RIGHT = "right";
+    static CENTER = "center";
+  }
```

Comment:
It might be advisable to split this out into a file or place it into a separate file?  Do we use the idea of Alignment with the same set of variables anywhere else in the code base?

Also, I think we should put this class at the top of the file?